.chalice/config.json

```diff
@@ -27,5 +36,9 @@
   "environment_variables": {
     "DATA_OBJ_INDEX": "fb_index",
     "DEBUG": "False"
+  },
+  "tags": {
+    "Owner": "davidcs@ucsc.edu",
+    "PURPOSE": "username=nalao:project=DataBiosphere/dos-azul-lambda:coordinator=casloan@ucsc.edu"
```

Comment:
@achave11 does this look right?

Reply:
@cricketsloan perhaps prefers different tags?